src/components/TutorialController.jsx

```diff
@@ -0,0 +1,50 @@
+import * as CONSTANTS from "../constants/constants";
+import Button from "./Button";
+
+const TutorialController = ({
+  orderButton,
+  isComplatedIndex,
```

Comment:
오타가 있는거 같습니다.

```suggestion
  isCompletedIndex,
```

isComplatedIndex => isCompletedIndex로 수정 부탁드립니다. 여러 곳에서 쓰이고 있을텐데 여기만 코멘트 달아두겠습니다.

Reply:
코드 리뷰 내용 반영했습니다! 감사합니다!